builder.go

```diff
@@ -547,6 +575,13 @@ func (b *UpdateBuilder) Where(expr BoolExpr) *UpdateBuilder {
 	return b
 }
 
+// Comments sets comments for the statement, replacing any
+// existing comments.
+func (b *UpdateBuilder) Comments(comments []string) *UpdateBuilder {
```

Comment:
Just checking but it doesn't seem like there are any `Upsert` builders right? Even though squalor has an `Upsert` command